### DISCUSSION TITLE
why-isnt-my-aspnetcore-app-in-docker-working/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#89] [Announcements]
# why-isnt-my-aspnetcore-app-in-docker-working/

In this post I describe a problem where my ASP.NET Core app in Docker wasn't responding to requests. This post debugs and diagnoses the problem

https://andrewlock.net/why-isnt-my-aspnetcore-app-in-docker-working/

**Comment by MichalGrzegorzak** [comment]
Why CentOS?

**Reply by andrewlock** [reply]
CentOS 7 uses an older version of glibc than some others like ubuntu, which means we need to build on CentOS 7 if we want to support it (we do!)

**Comment by PawelG-WWA** [comment]
That's the solution I was looking for. Thanks for an explanation, not only you gave an answer, but you described the process I was going through, that's why I was reading everything from the begging to an end with a smile :)

[a comment by an automated account is not shown]

**Comment by smdrager** [comment]
This helped me out. Thanks for posting!

**Comment by montoya8201** [comment]
Thank you very much, that was what I was needing...

**Comment by neu7ron2** [comment]
Thanks for putting this together, It took me ages to try and find out .net wasn't working but a python API was fine. My first clue was when I enabled debugging, as you can see from your screenshot in your blog it shows "Unable to bind to http://localhost:5000 on the IPv6 loopback interface:" ... which made we start to wonder about the binding.

**Comment by paramesh-eng** [comment]
Superb,,, It worked!!
